doc/source/formats/reference/imd.rst

```diff
@@ -0,0 +1,330 @@
+.. -*- coding: utf-8 -*-
+.. _IMD-format:
+
+==================================================================
+IMD (Data streamed via Interactive Molecular Dynamics protocol v3)
+==================================================================
+
+.. include:: classes/IMD.txt
+
+Real-time streaming of simulation data between molecular dynamics engines and receiving clients can be achieved using IMD protocols like IMDv2 and IMDv3. The :class:`~MDAnalysis.coordinates.IMD.IMDReader` implements the IMDv3 protocol, enabling live streaming of ongoing simulation data.
+
+.. note::
+   MDAnalysis supports **IMDv3 only**, which provides continuous, gap-free streaming and is implemented in modern versions of GROMACS, LAMMPS, and NAMD. IMDv2, while widely available, was designed primarily for visualization and allows gaps in the data stream.
+
+What is Streaming?
+==================
+
+Streaming involves processing data in real-time as it is generated, rather than storing it for later analysis. In molecular dynamics, this means sending simulation data to a client on-the-fly while the simulation is running, without writing large trajectory files to disk.
+
+This can be achieved through a TCP/IP socket connection between the simulation engine and receiving client, transmitting coordinates, velocities, forces, energies, and timing information using the IMDv3 protocol.
+
+MDAnalysis's :class:`~MDAnalysis.coordinates.IMD.IMDReader` uses the `imdclient <https://imdclient.readthedocs.io/>`_ package and provides a familiar interface for reading streaming data, similar to other trajectory readers in MDAnalysis.
+
+When to Use Streaming?
+======================
+
+Streaming analysis is particularly valuable for:
+
+**Long-running simulations**
+    Early detection of problems (crashes, artifacts, equilibration issues) can save computational resources.
+
+**Adaptive sampling workflows**
+    Real-time analysis can guide simulation parameters or trigger enhanced sampling methods.
+
+**Interactive research**
+    Immediate feedback allows researchers to make informed decisions about continuing, modifying, or terminating simulations.
+
+**Storage-constrained environments**
+    Analyze data as it's generated without storing large trajectory files.
+
+Installation and Setup
+======================
+
+Required Dependencies
+---------------------
+
+The IMDReader requires the ``imdclient`` package:
+
+.. code-block:: bash
+
+   pip install imdclient
+
+.. note::
+   MDAnalysis requires ``imdclient >= 0.2.2`` for its current implementation.
+
+MD Engine Configuration
+-----------------------
+
+We provide below example configurations for enabling IMDv3 streaming in popular MD engines.
+
+**GROMACS**
```

Comment:
Make this a heading of the right level.

It's bad practice to use emphasis markup instead of semantic markup for structuring documents. Not only does it make formatting inconsistent (using styles) but it also makes documents hard to read on screen readers and thus decreases the accessibility dramatically.